parquet/src/main/java/org/apache/iceberg/parquet/ParquetAvro.java

```diff
@@ -298,6 +301,11 @@ public Schema map(Schema map, Schema value) {
       return map;
     }
 
+    @Override
+    public Schema variant(Schema variant, Schema metadataResult, Schema valueResult) {
```

Comment:
This class fixes up Avro schemas when using ParquetAvro. I think the other changes that move to Iceberg generics for testing (rather than Avro generics) remove the need for this, but I'm including it because there's no harm.